configure.ml

```diff
@@ -1069,6 +1069,12 @@ let write_dbg_wrapper f =
   Unix.chmod f 0o555
 
 let _ = write_dbg_wrapper "dev/ocamldebug-coq"
+(* next line gets Exception: Unix.Unix_error (Unix.ENOENT, "execv", "/usr/bin/echo").
+let _ = Unix.execv "/usr/bin/echo" [|"HI MOM"|] *)
```

Comment:
Probably your echo is at /bin/echo (mine is at least, on debian).

Reply:
`which echo` in shell to check

Reply:
No, my echo is in a different place than yours.  No doubt "tr" and "mv" are not in the same directory across all platforms, either.  So Unix.execv is probably not a good choice for what I want to do.  Isn't there some library routine that accepts a bash command line?

```
$ which echo
/usr/bin/echo
```

Although with cygwin, "root" is a bit of an odd notion.  /cygdrive/c refers the to the entire Windows C: drive; from the perspective of Windows, /usr is a subdirectory in the cygwin installation.  It would be perverse if I had to give

Reply:
I suppose I could put the tr command into "configure", which is already a shell command.  But seems like there should be a way to do this in configure.ml.

Reply:
Indeed, the method requires the Windows path, like this:

```
let _ = Unix.execv "C:\\OCaml64\\bin\\echo" [|"ignore HI MOM\n"|]
```

which prints "HI MOM" and omits "ignore" (probably as a convenience for the caller).